ledger/participant-integration-api/src/main/scala/platform/apiserver/error/ErrorInterceptor.scala

```diff
@@ -96,6 +101,12 @@ final class ErrorInterceptor extends ServerInterceptor {
     )
   }
 
+  private def isInternalErrorThrownFromScalapbGrpcCompleteObserver[RespT, ReqT](status: Status) = {
```

Comment:
```suggestion
  private def isUnsanitizedInternalOrUnknown[RespT, ReqT](status: Status) = {
```

Reply:
How about handling the `UNKNOWN` in this method as well? After all, they convey the same thing albeit expressed differently depending on which lib generates them.

Reply:
My concern is that the original predicate is already somewhat unintuitive (as it tries to detect certain cases by making assumptions about scalapb implemention and our error categories) and now we're adding a conditional for a different second case. I tried to make the code reflect that we have those two distinct cases

Reply:
ok, fine with not changing but please use at least a shorter name for the internal method e.g. `isUnsanitizedInternal`

Reply:
Ok, done